src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -9,20 +9,13 @@
     <RollForward>LatestPatch</RollForward>
   </PropertyGroup>
 
+  <ItemGroup>
+    <FrameworkListFileClass Include="WindowsFormsIntegration.resources.dll" />
```

Comment:
Add a comment about why this is this way. Point out the target file where it is manually brought in.

Reply:
As you did above. :) Went backwards.